app/src/main/java/org/projectbuendia/client/sync/SyncAdapter.java

```diff
@@ -103,14 +69,23 @@
      * Select a phase by setting a boolean value of true for the appropriate key.
      */
     public enum SyncPhase {
-        SYNC_USERS,
-        SYNC_LOCATIONS,
-        SYNC_CHART_ITEMS,
-        SYNC_CONCEPTS,
-        SYNC_PATIENTS,
-        SYNC_OBSERVATIONS,
-        SYNC_ORDERS,
-        SYNC_FORMS
+        SYNC_USERS(R.string.syncing_users, new UsersSyncPhaseController()),
```

Comment:
I agree with you about the `Controller` thing - it's a filler word that I used because I didn't have a better word. In its current form, this is more of a `SyncPhaseRunnable`. I will rename it to that.

I think it's better as a narrower interface though - the existing enum handles the registration for us, and I think it's good to keep meta-information about the sync phases separate from the logic of the sync phases.